### DISCUSSION TITLE
Certs Dumper only generating .pem but not .crt & .key

### DISCUSSION THREAD

**Discussion by dwishaventures** [kereis/traefik-certs-dumper#43] [Q&A] [answered]
Hi, 

In my case it only dump key.pem and cert.pem 

How can I get server.crt & server.key

Any environment to set ? 

  certdumper:
    image: humenius/traefik-certs-dumper
    container_name: traefik_certdumper
    restart: unless-stopped
    volumes:
          # mount the folder which contains Traefik's `acme.json' file        
      - ./data:/traefik:ro
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - ./data/assets/ssl:/output:rw    
     
Thank you. 
Dwija

**Comment by kereis** [comment, marked as answer]
Hey,

this Docker image only extracts the public and private key of one domain's certificate. So currently, there is no way of extracting the server's private key.
